### ISSUE TITLE
Autocomplete with freeSolo - error in formik-material-ui-lab 0.0.4

### ISSUE THREAD

**Issue by danie7k** [stackworx/formik-material-ui#201]
Hi,
Typing a single character when using autocomplete (formik-material-ui-lab 0.0.4) with freeSolo throws:

> TypeError
> value.map is not a function
> Autocomplete@/node_modules/material-ui/lab/esm/Autocomplete/Autocomplete.js:262:30

After switching to formik-material-ui-lab 0.0.2 form works as expected.

Error reproduction:
https://codesandbox.io/s/wonderful-galois-hycjl

**Comment by cliedeman** [comment]
@danie7k Thanks I never tested freeSolo with multiple.

There are some issues with [freeSolo in general](https://github.com/stackworx/formik-material-ui/issues/197) I need to take a look at

**Comment by strmer15** [comment]
This seems to be some kind of problem with how the `multiple` and `freeSolo` props are being handled by `fieldToAutocomplete`. A workaround I found is to do something like this:

```
export function MultiAutocomplete<
  T,
  DisableClearable extends boolean | undefined
>({
  ref,
  ...props
}: AutocompleteProps<T, true, DisableClearable, true> & {
  ref?: React.Ref<HTMLInputElement>
}): React.ReactElement {
  return (
    <MuiAutocomplete
      {...fieldToAutocomplete(props)}
      ref={ref}
      freeSolo
      multiple
    />
  )
}
```

**Comment by cliedeman** [comment]
Does anyone have a sample of freeSolo working with Multiple?

I have an example here https://codesandbox.io/s/funny-shockley-3pnzb but can't get it to work

**Comment by strmer15** [comment]
> Does anyone have a sample of freeSolo working with Multiple?
> 
> I have an example here https://codesandbox.io/s/funny-shockley-3pnzb but can't get it to work

You had an incorrect `onChange` handler - here's a working example based on your component: https://codesandbox.io/s/youthful-pond-lyyw6?file=/src/Demo.js

There's also a working demo of it on the material-ui site, see https://material-ui.com/components/autocomplete/#multiple-values, the 3rd example.

**Comment by cliedeman** [comment]
@strmer15 thanks,

I added a test case and believe it has been fixed in 0.0.7

**Comment by strmer15** [comment]
Hey @cliedeman - thanks! I just tried it out and it's working for me now.
